vllm/attention/backends/abstract.py

```diff
@@ -299,6 +304,12 @@ def __init__(
     ) -> None:
         raise NotImplementedError
 
+    @staticmethod
+    def get_supported_block_size() -> list[int]:
+        # [0] is a placeholder: the actual block size will be determined
+        # by config.block_size at runtime.
+        return [0]
```

Comment:
Need a more careful design of this API. E.g., a backend that supports arbitrary block_size, or arbitrary large multiple of 16 (not sure which backend supports this).

I'm thinking of:
```
class MultipleOf:
     base: int

 def get_supported_block_size() -> list[Union[int, MultipleOf]]:
        # The supported block size of the attention backend
        # FIXME: regard all value as supported now. Need to update the attention backend to pass the real value.
        return [MultipleOf(1)]
```

CC @LucasWilkinson